### PR TITLE
feat: Add wrapping `springql` crate

### PR DESCRIPTION
**Pull request by kazuk**
## Issue number and link

Fixes: #192

## Describe your changes

- add wrapping crate `springql`
- re-export `springql-core` API's from `springql`

## Checklist before requesting a review

- [x] I follow the [Semantic Pull Requests](https://github.com/zeke/semantic-pull-requests) rules (bugfix/feature)
- [x] I specified links to related issues (must: bugfix, want: feature)
- [x] I have performed a self-review of my code (bugfix/feature)
- [ ] I have added thorough tests (bugfix/feature)
- [x] I have edited `## [Unreleased]` section in `CHANGELOG.md` following [keep a changelog](https://keepachangelog.com/en/1.0.0/) syntax (bugfix/feature)
- [x] I {made/will make} a related pull request for [documentation repo](https://github.com/SpringQL/SpringQL.github.io) (feature)

### CODE REVIEW

**Comment by laysakura** [comment]
[nits]

> [x] I follow the [Semantic Pull Requests](https://github.com/zeke/semantic-pull-requests) rules (bugfix/feature)

I changed the pull-request title to follow it.

**Review by laysakura — CHANGES_REQUESTED** [review]
- Want to move `tests/` and `examples/` from springql-core into springql, in order to keep E2E tests
- Want to eliminate duplication in springql-core and springql as much as possible.
  - `springql/src/lib.md` is duplicate of that of springql-core, for example.

**Comment by kazuk** [comment]
CI deadlink fail with #196

File: springql/src/lib.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
+<!-- markdownlint-disable MD041 -->
```
Comment (laysakura) [inline springql/src/lib.md:1]: Could you remove `springql-core/src/lib.md` ?

File: springql/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+// This file is part of https://github.com/SpringQL/SpringQL which is licensed under MIT OR Apache-2.0. See file LICENSE-MIT or LICENSE-APACHE for full license details.
+
+#![doc = include_str!("lib.md")]
+#![deny(missing_debug_implementations, missing_docs)]
+#![deny(rustdoc::broken_intra_doc_links)]
+
+pub use springql_core::api::*;
+
+/// error and result types for SpringQL
+pub mod error {
```
Comment (laysakura) [inline springql/src/lib.rs:10]: We see two `error` modules in doc.

- left: `cargo d` result
- right: https://docs.rs/springql-core/latest/springql_core/api/index.html

image

**Review by laysakura — CHANGES_REQUESTED** [review]

File: springql/src/lib.rs (hunk 2 above)
Comment (kazuk) [reply springql/src/lib.rs:10]: seems mod `error` from 

- `springql::error` and (`pub mod error` in springql`)
- `springql-core::api::error` (re-exported `pub use springql_core::api::*`)

**Review by laysakura — APPROVED** [review]
💥 🙏 💥

**Review by laysakura — CHANGES_REQUESTED** [review]
Ah, sorry. Please fix CI errors.

**Comment by laysakura** [comment]
📝 example-run is failing due to this issue: https://github.com/SpringQL/SpringQL/issues/204

**Review by laysakura — APPROVED** [review]

**Pull request merged**